ios/Classes/GigyaSdkWrapper.swift

```diff
@@ -688,6 +688,72 @@ extension GigyaSdkWrapper {
 
         result(nil)
     }
+
+    /**
+     Biometrics.
+     */
+
+    func isLocked(result: @escaping FlutterResult) {
+        guard let sdk = self.sdk else {
+            result(FlutterError(code: PluginErrors.missingParameterError, message: 'SDK is not initialised', details: nil))
+            return
+        }
+        
+        result(sdk.biometric.isLocked)
+    }
+    
+    func isOptIn(result: @escaping FlutterResult) {
+        guard let sdk = self.sdk else {
+            result(FlutterError(code: PluginErrors.missingParameterError, message: 'SDK is not initialised', details: nil))
+            return
+        }
+        
+        result(sdk.biometric.isOptIn)
+    }
+    
+    func optIn(result: @escaping FlutterResult) {
+        guard let sdk = self.sdk else {
+            result(FlutterError(code: PluginErrors.missingParameterError, message: 'SDK is not initialised', details: nil))
+            return
+        }
+        
+        sdk.biometric.optIn(completion: { biometricResult in
+            result(biometricResult == GigyaBiometricResult.success ? true : false)
+        })
+    }
+    
+    func optOut(result: @escaping FlutterResult) {
+        guard let sdk = self.sdk else {
+            result(FlutterError(code: PluginErrors.missingParameterError, message: 'SDK is not initialised', details: nil))
+            return
+        }
+        
+        sdk.biometric.optOut(completion: { biometricResult in
+            result(biometricResult)
```

Comment:
Should this be:

```suggestion
            if (biometricResult == GigyaBiometricResult.success) {
              result(nil)
            } else {
              result(FlutterError(...))
            }
```

as the biometric result is an enum, not a boolean.

Same remark for `lockSession` below

Reply:
I will go ahead and use ` result(biometricResult == GigyaBiometricResult.success ? true : false)` as I have done in the other functions so the error can be handled on the flutter side if GigyaBiometricResult.failure is returned